### PR TITLE
Update Constants classes with values and issues from Week 0 

### PR DESCRIPTION
**Pull request by kchenturtles**
Added drivetrain motor ID constants as well
(also update wpilib version to 4.1)

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -14,9 +14,9 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static final double SIMPLE_AUTON_SPEED = 0.2; // PLACEHOLDER
-  public static final double SIMPLE_AUTON_RUNTIME = 5.0; // PLACEHOLDER
-  public static final double MEDIUM_AUTON_OUTAKE_RUNTIME = 5.0; // PLACEHOLDER
+  public static final double SIMPLE_AUTON_SPEED = 0.5; // WEEK0
+  public static final double SIMPLE_AUTON_RUNTIME = 4.0; // WEEK0
+  public static final double MEDIUM_AUTON_OUTTAKE_RUNTIME = 2.0; // PLACEHOLDER
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/Constants.java:19]: Should be `// WEEK0`??

Comment (kchenturtles) [reply src/main/java/frc/robot/Constants.java:19]: I put that there to make it clearer what we've tested before and what we haven't
We'll still probably need to change motor ids and etc but hopefully not buttons/runtime/speed/current?
I think that answers the question?

**Review by chit-sudeep — DISMISSED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -66,7 +69,7 @@
       TowerConstants.BOTTOM_LEFT_TOWER_MOTOR_ID,
       TowerConstants.TOP_RIGHT_TOWER_MOTOR_ID,
       TowerConstants.BOTTOM_RIGHT_TOWER_MOTOR_ID);
-  private final Climber climber = new Climber(ClimberConstants.CLIMBER_MOTOR_ID);
+  private final Climber climber = new Climber(ClimberConstants.LEFT_CLIMBER_MOTOR_ID);
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/RobotContainer.java]: Conflicts with code from #73, discard this change

Comment (aidan-mundy) [inline src/main/java/frc/robot/RobotContainer.java:72]: Conflicts with code from #73

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -96,12 +101,13 @@
    * Stores constants related to the Climber.
    */
   public final class ClimberConstants {
-    public static final int CLIMBER_MOTOR_ID = 1; // PLACEHOLDER
-    public static final double CLIMBER_SPEED = 1.0; // PLACEHOLDER
+    public static final int LEFT_CLIMBER_MOTOR_ID = 50; // WEEK0
+    public static final int RIGHT_CLIMBER_MOTOR_ID = 51; // WEEK0
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/Constants.java:105]: Conflicts with #73, discard this change

Comment (aidan-mundy) [inline src/main/java/frc/robot/Constants.java]: Conflicts with #73, discard this change

**Review by aidan-mundy — CHANGES_REQUESTED** [review]

**Review by chit-sudeep — APPROVED** [review]

**Review by aidan-mundy — APPROVED** [review]

**Pull request merged**